platform/linux-generic/odp_schedule_basic.c

```diff
@@ -1249,6 +1249,17 @@ static uint64_t schedule_wait_time(uint64_t ns)
 	return ns;
 }
 
+static int schedule_capability(odp_schedule_capability_t *capa)
+{
+	memset(capa, 0, sizeof(odp_schedule_capability_t));
+
+	capa->max_ordered_locks = sched_fn->max_ordered_locks();
+	capa->max_groups = sched_fn->num_grps();
+	capa->prios = odp_schedule_num_prio();
```

Comment:
To avoid confusion, it's better to call these function directly (not through function pointer). For this file it means direct calls to schedule_max_ordered_locks(), schedule_num_grps() and schedule_num_prio().